lib/grandpa/message_handler.go

```diff
@@ -477,9 +522,13 @@ func (s *Service) VerifyBlockJustification(hash common.Hash, justification []byt
 		if !ok {
 			return ErrInvalidSignature
 		}
+
+		count++
 	}
 
-	if len(fj.Commit.Precommits)+equivocatoryVotersCount <= (2 * len(auths) / 3) {
+	// uses the current set of authorities to define the threshold
+	threshold := (2 * len(auths) / 3)
+	if count+len(eqvVoters) <= threshold {
```

Comment:
nit as a newbie, I prefer the `equivocatoryVotersCount` full name than `eqvVoters` 😄

Shall we just use longer explicit full names in the future as well? Thoughts anyone?

Reply:
Agree with explicit names, but in this case, I just need to use the `len` of the map of equivocatory voters, so I can change from `eqvVoters` to `equivocatoryVoters`.

Reply:
@EclesioMeloJunior I think this is still `eqvVoters` if I'm not mistaken, do you mind replacing it with `equivocatoryVoters`. I'm obsessed with full words sorry about that :sweat_smile: